tests/test_synced_collection.py

```diff
@@ -520,22 +546,30 @@ def test_update_recursive(self, synced_list):
         self.store(data1)
         assert synced_list == data1
 
-        # inavlid data in file
+        # invalid data in file
         data2 = {'a': 1}
         self.store(data2)
         with pytest.raises(ValueError):
             synced_list.load()
 
+        # reset the file
```

Comment:
This may be another example of the same problematic pattern with pytest fixtures / hypothesis. I commented on this specifically because it doesn't look like the other instances and would be easy to miss in a refactoring.